### ISSUE TITLE
There's a issue in searchable with new method model

### ISSUE THREAD

**Issue by holacliente** [ACFBentveld/Laravel-datatables#23]
When i tried searchables like:

DataTables::model(new Subscriber)
            ->searchable('batch.name', 'customer.razon_social','email','result')
            ->get();

i got all the results as same without filtering nothing..

is fixed with

return the entire model 

```php
private function searchOnModel()
{
    return $this->model->where(function ($query) {
        foreach ($this->searchable as $index => $column) {
            $this->searchOnRelation($column, $query);
            $this->searchOnQuery($column, $query, $index);
        }
    });
}
```

and writing execute in line 270 of Datatables.php

```php
protected function execute()
{
    $count = $this->model ? $this->model->count() : 0;

    if ($this->model && $this->search && $this->hasSearchable) {
        $this->model = $this->searchOnModel();
    }
}
```

**Comment by holacliente** [comment]
another thing, the pager is showing the complete results,

is fixed with next...

if ($this->model && $this->search && $this->hasSearchable) {
            $this->model = $this->searchOnModel();
            $count = $this->model->count();
        }

**Comment by wimurk** [comment]
This issue if fixed in version [v2.0.27](https://github.com/ACFBentveld/Laravel-datatables/releases/tag/v2.0.27)

**Comment by wimurk** [comment]
Datatables v3 is released [here](https://github.com/singlequote/Laravel-datatables). We are looking for contributors who can help us create an amazing package
